api/config_entry_gateways.go

```diff
@@ -0,0 +1,76 @@
+package api
+
+// IngressGatewayConfigEntry manages the configuration for an ingress service
+// with the given name.
+type IngressGatewayConfigEntry struct {
```

Comment:
`IngressGatewayConfigEntry` should have a namespace field, same with `IngressService`.

Here's an example for KV:
https://github.com/hashicorp/consul/blob/master/api/kv.go#L46

Reply:
Makes sense, updated.